script.js

```diff
@@ -189,17 +216,78 @@ function handleData([sheetsText, statesGeoJson]) {
     }
   });
 
+  countiesGeoJson = {
+    type: "FeatureCollection",
+    features: countiesCartoData.rows.map(
+      ({ the_geom, state, county }, index) => {
+        const properties = countiesData.filter(
+          row =>
+            row.municipality
+              .replace(" County", "")
```

Comment:
See my comment on using a spatial join for how we can avoid the inconsistent encodings of county names.
There's also a duplicate of `.replace(" County", "")` in this method chain.
This might be more efficient to do as a Regular Expression, but would have to look into it.